formatters/githubformat/githubformat.go

```diff
@@ -77,7 +96,19 @@ func (f *Formatter) Format(source string, i *manifest.Import, r manifest.Result)
 
 	for _, comment := range r.Comments {
 		fingerprint := fingerprint(source, comment)
-		if f.existingComments[fingerprint] {
+		if ec, ok := f.existingComments[fingerprint]; ok {
+			// This comment won't be marked as resolved because the checker still thinks it's a problem.
+			ec.Stale = false
+			 // Unresolve the comment if it was previously resolved
+			if ec.Type == github.FileComment {
+				if err := f.client.UnresolveFileComment(ec); err != nil {
+					return err
+				}
+			} else {
+				if err := f.client.UnresolveComment(ec); err != nil {
+					return err
```

Comment:
```suggestion
					return err
				}
			} else {
				if err := f.client.UnresolveComment(ec); err != nil {
					return err
```

Minor, and non-blocking. I wonder if we should `err` and continue instead of returning completely. That feels like a future enhancement (and drives a need for backoff+retry 😅)